.dotnet.azure/sdk/openai/Azure.AI.OpenAI/src/Custom/FineTuning/AzureCreateJobOperation.Protocol.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System.ClientModel;
+using System.ClientModel.Primitives;
+using System.Diagnostics.CodeAnalysis;
+
+#nullable enable
+
+namespace Azure.AI.OpenAI.FineTuning;
+
+/// <summary>
+/// A long-running operation for creating a new model from a given dataset.
+/// </summary>
+public class AzureCreateJobOperation : CreateJobOperation
+{
+    private readonly PipelineMessageClassifier DeleteJobClassifier = PipelineMessageClassifier.Create(stackalloc ushort[] { 204 });

```

Comment:
It would be good to lazily instantiate this.

Reply:
updated